releasenotes/notes/requests-b06e324e36e03ae5.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    requests: add global config support. This enables the requests service name to be
+    configured with ``ddtrace.config.requests['service']`` or the ``DD_REQUESTS_SERVICE``
```

Comment:
There's no `TRACE` prefix in this env var?

Reply:
We don't do it with our integrations: https://ddtrace.readthedocs.io/en/stable/integrations.html although there are some that we do DD_TRACE... 😕